webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2460,6 +2463,43 @@ var CURRENT_GROUP_DATA={};
                 mck_text_box.addEventListener("keyup",function(){
                     _this.toggleMediaOptions(this);
 
+                });
+                $mck_text_box.keyup(function (){
+                    if(CHAR_CHECK){
+                        var warning_box = document.getElementsByClassName('mck-char-warning')[0];
```

Comment:
@hardikparashar pls check whether _ is the naming convention used for all type of variables, it might be getting used only for $ jquery /ui associated references.

Reply:
done

Reply:
resolved in new commit, please check